lib/chef/run_list/run_list_expansion.rb

```diff
@@ -140,8 +152,9 @@ def applied_role(role_name)
       end
 
       def expand_run_list_items(items, included_by="top level")
+
         if entry = items.shift
-          @run_list_trace[included_by.to_s] << entry.to_s
+          @run_list_trace[included_by.to_s] <<  entry
```

Comment:
Ok, I didn't see any code using it in chef-client. Wasn't sure of the policy on this, I could just add a new field to do this with and leave the old one, it just felt crufty.